Client/src/components/App.jsx

```diff
@@ -47,7 +45,7 @@ class App extends React.Component {
   }
 
```

Comment:
Change to 'api/restaurant'.  I think the ./ is a file path whereas api/restaurant is a route